src/headless/cli.ts

```diff
@@ -0,0 +1,114 @@
+import { Command } from 'commander';
+import { CLIOptions } from '../types/CLIOptions';
+import { DataLoader } from './dataLoader';
+import { runShadingSceneHeadlessChrome } from './headlessBrowser';
+
+// Turn file names into array
```

Comment:
Can you turn this into proper docstring (with the /**) so that the description of the function is shown in the IDE when hovering over it?